.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.19]
```

Comment:
Since this is a yaml file, `1.20` is recognized as a number (i.e. `1.2`). To fix this, please add single quotes around it like `['1.20']`.

Reply:
Oh! Alright, fixing it. Thanks for pointing it out!